arangod/IResearch/IResearchFeature.cpp

```diff
@@ -632,10 +632,14 @@ void IResearchFeature::Async::start() {
     << "started " << _pool.size() << " ArangoSearch maintenance thread(s)";
 }
 
-IResearchFeature::IResearchFeature(arangodb::application_features::ApplicationServer* server)
+IResearchFeature::IResearchFeature(
+    arangodb::application_features::ApplicationServer& server
+)
   : ApplicationFeature(server, IResearchFeature::name()),
     _async(std::make_unique<Async>()),
-    _running(false) {
+    _running(false),
+    _threads(0),
+    _threadsLimit(0) {
```

Comment:
are you using these values anywhere ?

Reply:
in the next pull request... they are for configuring the poll from command-line options